lib/schema_registry/client.rb

```diff
@@ -15,31 +16,32 @@ def initialize(code, message)
   ServerError = Class.new(SchemaRegistry::ResponseError)
 
   RESPONSE_ERROR_CODES = {
-    40401 => (SubjectNotFound           = Class.new(SchemaRegistry::ResponseError)),
-    40402 => (VersionNotFound           = Class.new(SchemaRegistry::ResponseError)),
-    40403 => (SchemaNotFound            = Class.new(SchemaRegistry::ResponseError)),
-    42201 => (InvalidAvroSchema         = Class.new(SchemaRegistry::ResponseError)),
-    42202 => (InvalidVersion            = Class.new(SchemaRegistry::ResponseError)),
-    42203 => (InvalidCompatibilityLevel = Class.new(SchemaRegistry::ResponseError)),
+    40_401 => (SubjectNotFound           = Class.new(SchemaRegistry::ResponseError)),
+    40_402 => (VersionNotFound           = Class.new(SchemaRegistry::ResponseError)),
+    40_403 => (SchemaNotFound            = Class.new(SchemaRegistry::ResponseError)),
+    42_201 => (InvalidAvroSchema         = Class.new(SchemaRegistry::ResponseError)),
+    42_202 => (InvalidVersion            = Class.new(SchemaRegistry::ResponseError)),
+    42_203 => (InvalidCompatibilityLevel = Class.new(SchemaRegistry::ResponseError)),
     409   => (IncompatibleAvroSchema    = Class.new(SchemaRegistry::ResponseError)),
-    403   => (UnauthorizedRequest       = Class.new(SchemaRegistry::ResponseError)),
-  }
+    403   => (UnauthorizedRequest       = Class.new(SchemaRegistry::ResponseError))
+  }.freeze
```

Comment:
I am seeing a bunch of unrelated style changes (also single vs. double quotes, indentation, etc). I prefer to leave these out of scope for this PR so that this PR is only about adding TLS support.

If you feel strongly about them, you can open a separate PR to get these things fixed.

Reply:
yea, I ran it though rubocop... I'l revert